cmake/SetupCodeChecks.cmake

```diff
@@ -83,15 +83,22 @@ if(CLANGQUERY_FOUND)
 endif()
 
 if(CLANGTIDY_FOUND)
+    # targets for verifying clang-tidy
     add_custom_target(clang_tidy_check)
     add_dependencies(${BLT_CODE_CHECK_TARGET_NAME} clang_tidy_check)
```

Comment:
Also wondering if `clang-tidy-check` should be run whenever we run `make check` since it can have a lot of false positives if you're not careful about reducing rules and possibly adding some `NOLINT*` lines to your code.

See: https://clang.llvm.org/extra/clang-tidy/index.html#suppressing-undesired-diagnostics

Reply:
I've removed `clang_tidy_check` from the main `check` target, per group discussion. See comments elsewhere on this PR.